shade-test/jar-file/src/test/java/org/testcontainers/JarFileShadingTest.java

```diff
@@ -61,7 +61,8 @@ public void testMetaInf() throws Exception {
         );
 
         assertThatFileList(root.resolve("META-INF").resolve("native")).containsOnly(
-                "liborg-testcontainers-shaded-netty-transport-native-epoll.so"
+                "liborg-testcontainers-shaded-netty-transport-native-epoll.so",
+                "liborg-testcontainers-shaded-netty-transport-native-kqueue.jnilib"
```

Comment:
@rnorth seems like this test actually helps :D We almost shipped unshaded native dependency :)